app/hall.hoon

```diff
@@ -916,6 +915,21 @@
     ::>    arms that make miscellaneous changes to this story.
     ::+|
     ::
+    ++  so-present                                      ::<  accept status diff
+      |=  {who/ship dif/diff-status}
+      ^+  +>
+      ::  only have presence if you have write permission.
+      ?.  |((so-admire who) ?=($remove -.dif))  +>
```

Comment:
If `who` sends a `$remove` command without write permissions, then shouldn't we ignore it or send back a permissions error? If I'm reading this code correctly, i we get a `$remove` from someone who does not have write permissions, we might still perform the removal.

Reply:
If someone has been banned, they still need to have their presence removed. Hence why the permissions check doesn't apply there.